components/google_drive/actions/find-folder/find-folder.mjs

```diff
@@ -25,10 +25,18 @@ export default {
       ],
       description: "The name of the folder to search for",
     },
+    includeTrashed: {
+      type: "boolean",
+      label: "Include Trashed",
+      default: true,
+      optional: true,
+      description: "If set to true, returns all matches including items currently in the trash."
```

Comment:
**ESLint**

Missing trailing comma

I think it is good to add the `default` value to the description too

```suggestion
      description: "If set to true, returns all matches including items currently in the trash. Defaults to `true`.",
```

Reply:
Fixed. I agree, it's better with the default explanation.